### ISSUE TITLE
kitchen sink animations broken?

### ISSUE THREAD

**Issue by rjleveque** [clawpack/riemann_book#86]
@ketch: The ianimates in `Kitchen_sink_problem.ipynb` seem to produce empty axes.  I haven't investigated...

**Comment by ketch** [comment]
They work fine for me (tested just now).  Must be something that changed in your setup?

**Comment by maojrs** [comment]
Having the same problem here, empty axes. I will try to look into it.

**Comment by ketch** [comment]
Maybe make sure you have updated visclaw.  No idea if that's the cause, though.

**Comment by ketch** [comment]
Did they work for you in the past?

**Comment by maojrs** [comment]
Just updated visclaw, same problem. Didn't try to run them in the past, so I am not sure if they worked before.

**Comment by rjleveque** [comment]
For me it works locally, it's only on Azure that it seems broken.

But now when I tried it again I'm having problems installing clawpack on Azure.  Can someone else try executing `Install-packages.ipynb` and then `Kitchen_sink_problem.ipynb` on Azure and see if you run into the same thing.  This worked before...

https://notebooks.azure.com/rjleveque/libraries/riemann-book

**Comment by ketch** [comment]
Clawpack doesn't install correctly for me on Azure (I get "no module named clawpack" when I try to import it after installing).

**Comment by ketch** [comment]
I restarted the notebook and ran it again and it works now.  And I do get blank animations in the Kitchen sink notebook.  I suspect that this is simply because Azure and JSAnimation are not playing well together.  Are there JSAnimations in other notebooks that work on Azure?

**Comment by rjleveque** [comment]
Right, Clawpack can't be imported in the notebook used to do the installation or other notebooks already open (unless you restart the kernel).

JSAnimation does work in `Introduction.ipynb` on Azure -- change the import widgets statement to
`from utils.jsanimate_widgets import interact` to test.
